### PR TITLE
Use the doctrine paramconverter

### PR DESCRIPTION
**Pull request by acrobat**
Use the doctrine paramconverter instead of manually fetching from db. This simplifies the code and moves more logic out of the controllers

### CODE REVIEW

File: src/Intracto/SecretSantaBundle/Controller/Participant/ParticipantController.php [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 
 use Intracto\SecretSantaBundle\Validator\ParticipantIsNotBlacklisted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
 use Intracto\SecretSantaBundle\Entity\Participant;
 use Intracto\SecretSantaBundle\Entity\EmailAddress;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\VarDumper\VarDumper;
```
Comment (tvlooy) [inline src/Intracto/SecretSantaBundle/Controller/Participant/ParticipantController.php]: unused use

File: src/Intracto/SecretSantaBundle/Controller/Party/ManagementController.php [hunk 2]
```diff
@@ -203,24 +191,21 @@ public function startPartyAction($listUrl)
...
 
     /**
-     * @Route("/exclude/{listUrl}", name="party_exclude")
+     * @Route("/exclude/{listurl}", name="party_exclude")
      */
-    public function excludeAction(Request $request, $listUrl)
+    public function excludeAction(Request $request, Party $party)
     {
-        /** @var MailerService $mailerService */
-        $party = $this->getParty($listUrl);
```
Comment (tvlooy) [inline src/Intracto/SecretSantaBundle/Controller/Party/ManagementController.php]: wow, ugly typehint we had there :-)

File: src/Intracto/SecretSantaBundle/Controller/Participant/ParticipantController.php (hunk 1 above)
Comment (acrobat) [inline src/Intracto/SecretSantaBundle/Controller/Participant/ParticipantController.php]: Fixed!

File: src/Intracto/SecretSantaBundle/Controller/Party/ManagementController.php (hunk 2 above)
Comment (acrobat) [inline src/Intracto/SecretSantaBundle/Controller/Party/ManagementController.php]: :smile:

**Comment by acrobat** [comment]
It's indeed a bit of magic, but no different than the `Request` typehint we already use. It removes some extra logic in the controllers so that was enough for me to move to the paramconverter :smile: 

Let's strip some more code out of the controllers :smile:

**Pull request merged**